gdal/gdal_macros.xml

```diff
@@ -0,0 +1,145 @@
+<macros>
+    
+    <xml name="gdal_requirements">
+        <requirements>
+            <requirement type="package" version="2.1.0">gdal</requirement>
```

Comment:
I would add a macro for the version ... then use this macro here and default to all tool versions to the same tool version. Like here: https://github.com/galaxyproject/tools-iuc/blob/master/tools/augustus/augustus.xml#L1

This has the nice effect that the tool version is always the gdal version.

Reply:
Did it and indeed its better this way